flink-end-to-end-tests/test-scripts/common_kubernetes.sh

```diff
@@ -21,26 +21,35 @@ source "$(dirname "$0")"/common.sh
 source "$(dirname "$0")"/common_docker.sh
 
 CONTAINER_SCRIPTS=${END_TO_END_DIR}/test-scripts/container-scripts
-MINIKUBE_START_RETRIES=3
-MINIKUBE_START_BACKOFF=5
+RETRY_COUNT=3
+RETRY_BACKOFF_TIME=5
 RESULT_HASH="e682ec6622b5e83f2eb614617d5ab2cf"
 MINIKUBE_VERSION="v1.28.0"
 
 NON_LINUX_ENV_NOTE="****** Please start/stop minikube manually in non-linux environment. ******"
 
+function get_arch() {
+  if [[ `uname -i` == 'aarch64' ]]; then
+    local arch='arm64'
+  else
+    local arch='amd64'
+  fi
+  echo "${arch}"
```

Comment:
```suggestion
    echo 'arm64'
  else
    echo 'amd64'
  fi
```
nit: we don't need the local variable here